pkg/odp/event/event_manager.go

```diff
@@ -83,7 +75,11 @@ func WithQueueSize(qsize int) EMOptionFunc {
 // default value is 1 second
 func WithFlushInterval(flushInterval time.Duration) EMOptionFunc {
 	return func(bm *BatchEventManager) {
-		if flushInterval > 0 {
+		if flushInterval >= 0 {
```

Comment:
```
bm.batchSize = flushInterval == 0 ? 1 : bm.batchSize
```

Reply:
ternary operators are not supported in golang.